CHANGELOG.md

```diff
@@ -6,6 +6,153 @@
 
 Releases are now named after famous [Cricket](https://en.wikipedia.org/wiki/Cricket) players.
 
+## [0.5] - [TBD] - 2017-11-??
+
+JupyterHub 0.8, HTTPS & scalability.
+
+### Upgrading from 0.4
+
+#### DB upgrade
+
+This release contains a big JupyterHub version bump (from 0.7.2 to 0.8). If
+you are using the default database provider (sqlite), then the required db upgrades
+will be performed automatically when you do a `helm upgrade`.
+
+If you are using MySQL / PostgreSQL as your backing database, you will need to
+perform a manual database upgrade with the following steps:
+
+1. Make a full backup of your database, just in case things go bad.
+2. Make sure that the database user used by JupyterHub to connect to your database
+   can perform schema migrations like adding new tables, altering tables, etc.
+3. In your config.yaml, add the following config:
+
+   ```yaml
+   hub:
+     db:
+       upgrade: true
+   ```
+4. Do a `helm upgrade`. This should perform the database upgrade needed.
+5. Remove the lines you added in step 3, and do another `helm upgrade`.
+
+It is highly recommended that you have a `staging` environment that matches
+your production environment & can be used as a testbed for this upgrade. Feel
+free to reach out to us on [gitter](https://gitter.im/jupyterhub/binder) before
+your upgrade if you have any questions!
+
+#### Ingress config incompatibilities
+
+We've made HTTPS much easier to set up, with automated certificates from
+[Let's Encrypt](https://letsencrypt.org/) integration. However, this means
+that some of the keys used to set up your own ingress has changed.
+
+If you were using config under `ingress` purely to get HTTPS, we recommend
+that you delete your entire config section under `ingress` & instead follow
+the new [docs](<- link -> ) on getting HTTPS set up. It's much easier & a lot
```

Comment:
TODO: link

Reply:
I looked but couldn't find the link.

Reply:
Because there aren't any yet!

Reply:
@willingc https://github.com/jupyterhub/zero-to-jupyterhub-k8s/pull/306 should do it